go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -302,7 +307,7 @@ func tearDown(t *testing.T, initMysql bool) {
 		_, err = tablet.VttabletProcess.QueryTablet(disableSemiSyncCommands, keyspaceName, true)
 		require.Nil(t, err)
 		for _, db := range []string{"_vt", "vt_insert_test"} {
-			_, err = tablet.VttabletProcess.QueryTablet(fmt.Sprintf("drop database if exists %s", db), keyspaceName, true)
+			_, err = tablet.VttabletProcess.QueryTabletWithSuperReadOnlyHandling(fmt.Sprintf("drop database if exists %s", db), keyspaceName, true)
```

Comment:
Here, we can drop the databases on the primary before we stop the replication. Then we can wait for the tablets to catch up and then stop replication everywhere. This way we wouldn't need to use the `QueryTabletWithSuperReadOnlyHandling` method.

Reply:
Ok I tried a different approach here..